src/registrar/templates/includes/domain_example.html

```diff
@@ -55,45 +54,49 @@
   </ul>
 
 {% elif organization_type == 'county' %}
-  <p>Most county .gov domains must include the two-letter state abbreviation or the full state name. County names that aren’t shared by any other city, county, parish, town, borough, village or equivalent in the U.S., at the time a domain is granted, can be requested without referring to the state. Counties can include “county” in their domain to distinguish it from other places with similar names. We use the <a class="usa-link usa-link--external" rel="noopener noreferrer" target="_blank" href="https://www.census.gov/geographies/reference-files/time-series/geo/gazetteer-files.html">Census Bureau’s National Places Gazetteer Files</a> to determine if county names are unique.</p>
+  <p>Most county .gov domains must include the two-letter state abbreviation or the full state name. County names that aren’t shared by any other city, county, parish, town, borough, village or equivalent in the U.S. (at the time a domain is granted) don’t have to refer to their state in their domain name. Counties can include “county” in their domain to distinguish it from other places with similar names.</p> 
+    
+<p>We use the <a class="usa-link usa-link--external" rel="noopener noreferrer" target="_blank" href="https://www.census.gov/geographies/reference-files/time-series/geo/gazetteer-files.html">Census Bureau’s National Places Gazetteer Files</a> to determine if county names are unique.</p>
   <p><strong>Examples:</strong></p>
   <ul class="usa-list">
-    <li>AdamsCountyMS.gov</li>
-    <li>Erie.gov</li>
+    <li>LACounty.gov</li>
     <li>LivingstonParishLA.gov</li>
     <li>MitchellCountyNC.gov</li>
+    <li>MiamiDade.gov</li>
   </ul>
 
 {% elif organization_type == 'city' %}
   <p>Most city domains must include the two-letter state abbreviation or clearly spell out the state name. Using phrases like “City of” or “Town of” is optional.</p>
   <p>Cities that meet one of the criteria below don’t have to refer to their state in the domain name.
```

Comment:
The google doc has a slight change here:

> Cities that meet one of the criteria below don’t have to refer to their state in **their** domain name.

Reply:
```suggestion
  <p>Cities that meet one of the criteria below don’t have to refer to their state in their domain name.
```